components/public-api-server/pkg/apiv1/oidc.go

```diff
@@ -66,33 +73,6 @@ func (s *OIDCService) CreateClientConfig(ctx context.Context, req *connect.Reque
 	}), nil
 }
 
-func papiConfigToIAM(c *v1.OIDCClientConfig) *iam.OIDCClientConfig {
```

Comment:
These are moved below the service functions. This is per go guidelines where the struct instance methods should remain together.